### ISSUE TITLE
Next Week Plan (03-19-21)

### ISSUE THREAD

**Issue by bagmk** [bagmk/Quantum_Machine_Learning_Express#3]
## Plan for next week

- ~Study the relationship of the number of the shots~
- ~Study the relationship of different simulators~
- Three different ways to characterize y value for loss functions (avg, single shot, most occurred value)
- ~Read the Paper carefully how they defined the loss and compare the expressibility numerically from the paper.~
- ~Sanity check of the circuit (single data, check the convergence)~
- Reconfirm the circuit is correct (one by one)
-  Understand the interface between Qiskit and PyTorch (continued)
-  Calling function from others (% run my file?)

## Meeting summary

- Relationship between number of shots and simulator
- Discussion of PyTorch application
- Possible way of the convergence of loss function
- Discussion of divergence and entangling capability
- The class conversion from density matrix to an array
- u3 gate and random_unitary
